src/bug-filing/github/create-gitHub-bug-filing-url.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
not sure the casing in this filename follows our pattern. the 'H' in git**H**ub caught my attention here